skbio/stats/composition.py

```diff
@@ -608,10 +610,271 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0")
+def ancom(mat, cats,
+          alpha=0.05,
+          multicorr=False,
```

Comment:
A few options that I can think of

`multiple_comparisons`

`multiple_comparisons_corrections`

`multiple_hypothesis_corrections`

Any preference?

On Wed, Oct 14, 2015 at 10:10 AM, Evan Bolyen <notifications@github.com>
wrote:

> In skbio/stats/composition.py
> <https://github.com/biocore/scikit-bio/pull/1114#discussion_r42022389>:
>
> >      return perturb_inv(mat, cen)
> >
> >
> > +@experimental(as_of="0.4.0")
> > +def ancom(mat, cats,
> > +          alpha=0.05,
> > +          multicorr=False,
>
> Is there a better name for this? My first thought without knowing anything
> was that multicorr was for multi-correlation not Bonferroni.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/1114/files#r42022389>.
>